pkg/openthread/contrib/conn/conn_udp.c

```diff
@@ -0,0 +1,143 @@
+/*
+ * Copyright (C) 2016 José Ignacio Alamos
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @{
+ *
+ * @file
+ * @author  José Ignacio Alamos <jialamos@uc.cl>
+ */
+
+#include "net/conn/udp.h"
+#include "openthread/conn.h"
+#include "stdio.h"
+#include <openthread.h>
+#include "string.h"
+#include <net/ipv6/addr.h>
+#include "ot.h"
+#include "errno.h"
+
+#define ENABLE_DEBUG (0)
+#include "debug.h"
+
+static msg_t _msg;
+
+static conn_udp_msg_t msg_info;
+static otSockAddr sockaddr = {};
+
+void HandleUdpReceived(void *aContext, otMessage aMessage, const otMessageInfo *aMessageInfo)
```

Comment:
if possible (e.g., this is not defining an openthread callback), please don't use camelcase for functions and variable names.